### PR TITLE
Add argument for start synapse unit test framework agent

### PR DESCRIPTION
**Pull request by sajithaliyanage**
## Purpose
Passing parameters to start the unit testing framework for synapse agent

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Comment by sajithaliyanage** [comment]
Removed **test** tag from the script file (integrator.sh) and changed unitTest tag to **synapseTest**.

File: distribution/src/scripts/integrator.bat [hunk 1]
```diff
@@ -1,181 +1,192 @@
-@echo off
-
-REM ---------------------------------------------------------------------------
-REM        Copyright 2005-2009 WSO2, Inc. http://www.wso2.org
-REM
-REM  Licensed under the Apache License, Version 2.0 (the "License");
-REM  you may not use this file except in compliance with the License.
```
Comment (sdkottegoda) [inline distribution/src/scripts/integrator.bat]: The file has a diff that is not related to your fix. Shall we only have the relevant diff?

Comment (sajithaliyanage) [reply distribution/src/scripts/integrator.bat]: Done

**Comment by sajithaliyanage** [comment]
After upgrading the synapse to version 2.1.7-wso2v103 please merge this PR.
